content/e/kpw/2025-11-14-kpw-arena-xxx.md

```diff
@@ -26,18 +26,83 @@ This is the third KPW show, following [Arena V](@/e/kpw/2017-01-14-kpw-arena-v.m
+- - '[Team Visegrad](@/tt/team-visegrad.md): [David Oliwa](@/w/david-oliwa.md) & [Ricky Sky](@/w/ricky-sky.md)(c)'
+  - '[Greg](@/w/greg.md) & [Ron Corvus](@/w/ron-corvus.md)'
+  - s: Tag Team Match
+    c: '[KPW Tag Team Championship](@/c/kpw-tag-team-championship.md)'
+- - '[David Oliwa](@/w/david-oliwa.md)'
+  - '[Ricky Sky](@/w/ricky-sky.md)'
+  - '[Zefir](@/w/zefir.md)'
+  - '[Chemik](@/w/chemik.md)'
+  - '[Leon Lato](@/w/leon-lato.md)'
+  - '[Michał Fux](@/w/michal-fux.md)'
+  - '[Filip Fux](@/w/filip-fux.md)'
+  - '[Eryk Lesak](@/w/eryk-lesak.md)'
+  - '[Fynn Freyhart](@/w/fynn-freyhart.md)'
+  - '[Krystian Malinowski](@/w/krystian-malinowski.md)'
+  - g: '10th anniversary celebration; Malinowski fires Zefir.'
+- credits:
+    Referee: '[Krystian Czekaj](@/w/krystian-czekaj.md)'
+    Ring announcer: '[Kinga Miotke](@/w/kinga-miotke.md)'
 {% end %}
 
 ### Recap
-* TBD
+* [Kinga Miotke](@/w/kinga-miotke.md) opened the show and introduced [Zefir](@/w/zefir.md), who came out and cut a lengthy promo stating that after proving himself and winning the [KPW Championship](@/c/kpw-championship.md), he now has something to say and that he intends for his name to be forever associated with the title.
+* [Leon Lato](@/w/leon-lato.md) then came out to the ring and told Zefir "to his face… or rather, to his mask" that their fight had nothing to do with their friendship, but rather it was about proving who's the best. He then told Zefir to be ready for him, to which Zefir answered that he was - and since Leon was as well, the two agreed to fight right away.
+* [Krystian Czekaj](@/w/krystian-czekaj.md) was greeted with ["sędzia chuj"](@/a/polish-wrestling-chants.md#referee) chants as soon as Zefir asked him into the ring.
+* During the match, Erik Asmus and Friedrich Pohlmann ([Die Ordnung 2.0](@/tt/die-ordnung.md), who had been sitting among the audience members) ran into the ring and attacked the contenders, causing the match to end in a DQ in Leon's favour. They were soon chased off by [Filip Fux](@/w/filip-fux.md) and [Michał Fux](@/w/michal-fux.md). Afterwards, [Hans Schulte](@/w/hans-schulte.md) appeared on the screen and congratulated his countrymen for a job well done, calling them "my sons" (_meine Söhne_).
+* [Eryk Lesak](@/w/eryk-lesak.md) then came into the ring carrying his championship contract he won at [Godzina Zero 2025](@/e/kpw/2025-08-22-kpw-godzina-zero-2025.md) and said that had it not been for the Germans, he would be the next owner of both titles. Rookie Three (standing outside the ring but told by Lesak to come inside because so that he could hear him better) corrected him by pointing out that the contract is only valid for one championship title. He then added that Lesak had missed his chance by taking too long, which angered Eryk and prompted him to challenge Rookie Three to a fight. Victorious, Lesak left the arena carrying unconscious Rookie Three over his shoulder.
```

Comment:
```suggestion
* [Eryk Lesak](@/w/eryk-lesak.md) then came into the ring carrying his championship contract he won at [Godzina Zero 2025](@/e/kpw/2025-08-22-kpw-godzina-zero-2025.md) and said that had it not been for the Germans, he would be the next owner of both titles. Rookie Three (standing outside the ring but summoned inside by Lesak to come inside so that he could hear him better) corrected him by pointing out that the contract is only valid for one championship title. He then added that Lesak had missed his chance by taking too long, which angered Eryk and prompted him to challenge Rookie Three to a fight. Victorious, Lesak left the arena carrying unconscious Rookie Three over his shoulder.
```